<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankPage</class>
 <widget class="QWidget" name="BankPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="top">
     <item>
      <widget class="QPushButton" name="BackToWelcomePage">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollAreaUnprocessTransaction">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetUnprocessTransaction">
      <layout class="QHBoxLayout" name="unprocessTransaction"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollAreaActiveTransaction">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetActiveTransaction">
      <layout class="QHBoxLayout" name="activeTransaction"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollAreaTransactionProcessed">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetTransactionProcessed">
      <layout class="QHBoxLayout" name="transactionProcessed"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
